### PR TITLE
Update all.json

### PR DESCRIPTION
**Pull request by dubstard**
<html>
<body>

Site | URLScan link
-- | --
coinhamsterdrop.com |https://urlscan.io/result/9500f705-a82e-4590-ad18-719d12d84608/dom/
empiretokenworld.com|https://urlscan.io/result/84d3c5a7-ea03-499e-a04d-84c5a2cce2fd/dom/
pro-nfts.net|https://user-images.githubusercontent.com/49607867/163221084-d2ade4b0-6dad-4b95-8ec4-fae6829ec5ed.png
coinsencrypt.com|https://user-images.githubusercontent.com/49607867/163222901-306b030a-399f-442f-8ca7-49adae2fab40.png
connectvalidateit.com|https://user-images.githubusercontent.com/49607867/163221260-c4870a1e-d17f-459a-b517-d142bd6aa9c1.png
dappn-wallets.com|https://user-images.githubusercontent.com/49607867/163223504-056e04ad-0136-45f9-8146-69a15555dad0.png
speed-digital.app|https://user-images.githubusercontent.com/49607867/163225264-f86d4e56-e6df-40b7-ac24-a77ba46dbb08.png

</body>
</html>

"[coinhamsterdrop.com](https://urlscan.io/result/9500f705-a82e-4590-ad18-719d12d84608/)",
image

"[empiretokenworld.com](https://urlscan.io/result/84d3c5a7-ea03-499e-a04d-84c5a2cce2fd/)",
image

"pro-nfts.net",
image

"connectvalidateit.com",
image

"coinsencrypt.com",
image

"dappn-wallets.com",
image

"speed-digital.app",
image

```
"cake-defi.net",
"coinhamsterdrop.com",
"collab-land-verify.com",
"collablandconnect.net",
"collablandsync.net",
"collablandupdate.com",
"dapps-solution.com",
"dapps-solution.com.rectificationwal.online",
"empiretokenworld.com",
"onetaptap.online",
"verify.cake-defi.net",
"walletlogin.info",
"bep2.online",
"coinsencrypt.com",
"coinsynconline.com",
"connectvalidateit.com",
"dappn-wallets.com",
"ophiophagous-smiley-unvascular.s3.eu-central-003.backblazeb2.com",
"pro-nfts.net",
"registry.walletbackup.co",
"speed-digital.app",
"supperwallet.com",
"walleterrorresetore.com",
"walletsynchronization.online",
```

### CODE REVIEW

File: all.json [hunk 1]
```diff
@@ -25,6 +25,18 @@
...
 		"2polkadot.com",
 		"2rfleche.ma",
+		"cake-defi.net",
+		"coinhamsterdrop.com",
+		"collab-land-verify.com",
+		"collablandconnect.net",
+		"collablandsync.net",
+		"collablandupdate.com",
+		"dapps-solution.com",
+		"dapps-solution.com.rectificationwal.online",
```
Comment (jacogr) [inline all.json:35]: This seems listed? `rectificationwal.online` is blocked.

Comment (dubstard) [reply all.json:35]: ah, true will remove

**Review by dubstard — COMMENTED** [review]
"dapps-solution.com.rectificationwal.online", removing as we have blocked the main domain anyway "rectificationwal.online",

**Comment by dubstard** [comment]
thank you

**Pull request merged**